pymc/distributions/timeseries.py

```diff
@@ -250,10 +250,10 @@ def random(self, point=None, size=None):
 
         Parameters
         ----------
-        point: dict, optional
+        Point : dict, optional
```

Comment:
```suggestion
        point : dict or Point, optional
```

the parameter is point, the type is Point, it can be confusing. parameters are at the left of the colon, types on the right

Reply:
Thanks for explaining and for the rest of your comments!